packages/hashi/README.md

```diff
@@ -173,3 +173,11 @@ H5P Static Files
 This code is currently generated from https://github.com/h5p/h5p-php-library
 
 To update, update the `h5pCommit` variable in `downloadH5PVendor.js` to the desired tag and then run `yarn run build-h5p`.
+
+
+Bloom Reader Static Files
+-------------------------
+
+This code is currently generated from https://github.com/learningequality/bloom-player (specifically the 'patched' default branch).
+
+To regenerate, the repository should be cloned, and `yarn run build` run within the context of that repository to regenerate the new assets. All the files put into `dist` should then be copied into `kolibri/core/content/static/bloom` in the Kolibri repository. Any previously existing hash named files can be deleted and replaced by the new hash named files.
```

Comment:
this might be somewhat out of scope for the readme, but what are the scenarios when we would want to regenerate the static files?

Reply:
When there is an update from the bloom repo. Once I have finished the auto-updating H5P action and got that working, I am going to make one for bloom too.